zuul.d/podified_multinode.yaml

```diff
@@ -15,15 +15,6 @@
     nodeset: centos-9-medium-crc-extracted-2-30-0-3xl
     run:
       - ci/playbooks/edpm/run.yml
-    irrelevant-files:
-      - ^roles/.*_build
-      - ^roles/build.*
-      - ^roles/local_env_vm
-      - ^ci/templates
-      - ^docs
-      - ^.*/*.md
-      - ^OWNERS
-      - ^.github
```

Comment:
add to this list test/functional don't remove it

Reply:
@marios Thank you for the review, This job cifmw-base-multinode-podified is parented from cifmw-podified-multinode-edpm-base-crc. cifmw-podified-multinode-edpm-base-crc job uses the[ `irrelevant-files: *ir_files`](https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/base.yaml#L121) defined in [content provider job irrelevant file ](https://github.com/openstack-k8s-operators/ci-framework/blob/main/zuul.d/base.yaml#L30) .

The above irrelevant file list defined under cifmw-base-multinode-podified is already included in content provider job. That's why we are removing it.

Reply:
ack as discussed in slack we were missing the noop. indeed this irrelevant files match is working as intended (via inheritance) and the job freeze thing went away.